packages/cli/internal/pkg/cli/config/config.go

```diff
@@ -6,7 +6,10 @@ type User struct {
 	// so nobody would be tempted to change that in the config file and get inconsistency in between email and userid
 	Id string `yaml:"-"`
 }
-
+type Format struct {
```

Comment:
Could you please explain purpose of this struct? What is the benefit of using it versus just having a string field? We can consider creating a type like `type Format string`.

Reply:
The way the output displays seems to rely on the struct structure. By just adding format as string field in the Config struct, the 'configure describe' command yields an output like this.  
```
CONFIG	text
USER	atahlil@amazon.com	atahlil3HYhj7
```

Where as adding the struct would yield an output like this 
```
CONFIG	
FORMAT    text
USER	atahlil@amazon.com	atahlil3HYhj7
```
I am not sure if there is a better way to construct this without compromising the way that the output will display